calblueprint/meepanyar
# Created User Profile Page and Logout

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
This PR starts building out the Profile pages according to the Figma. 

This PR does the following:
1. Transfers the logout functionality out of `BaseHeader`
2. Creates a `/profile` endpoint that houses both the `My Information` `Site Information` and `Logout`
3. Transfers logout functionality to the `/profile` page
4. Log out now works even if offline.
3. Creates a `/profile/user` endpoint that houses the "My Information" stuff

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
1. Open the app and log in
2. Click the `Profile` button on the top right of the home screen to navigate to `/profile`
3. Verify that the `Profile` page and `My Information` pages are identical to the Figma
4. Perhaps advise on any CSS best practices that i might be violating if applicable?
5. Try the log out and verify that it works

[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps
1. Turn the "Profile" icon to the correct icon
2. Swap out outline button for base component outline button when Annie's base component [PR](https://github.com/calblueprint/meepanyar/pull/80) gets merged
3. Make functional changes (be able to edit user information)

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots
![image](https://user-images.githubusercontent.com/35501399/114277323-8ef35300-99df-11eb-9c1e-359d1bbe9907.png)

![image](https://user-images.githubusercontent.com/35501399/114260544-6b51ed80-998a-11eb-963c-db77d6d69717.png)

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
